weblate/utils/fields.py

```diff
@@ -110,3 +110,7 @@ def formfield(self, **kwargs):
                 **kwargs,
             }
         )
+
+class CommitEmailField(EmailField):
+    description = _("Commit e-mail")
+    pass
```

Comment:
This class seems not useful, you override description in the `User` class anyway.